themes/src/main/resources/theme/keycloak-preview/account/messages/messages_en.properties

```diff
@@ -68,14 +68,16 @@ notSetUp={0} is not set up.
 two-factor=Two-Factor Authentication
 passwordless=Passwordless
 unknown=Unknown
+password-display-name=Password
+password-help-text=Log in by entering your password.
```

Comment:
Our standard is to use "Sign in/Sign Out" instead of "Log in/Log Out".  But I see we've messed this up in a lot of places.